src/components/CountMovie/CountMovie.tsx

```diff
@@ -0,0 +1,10 @@
+import styles from './CountMovie.module.scss';
+
+interface countMovieProps {
+  countMovie: number;
+}
+
```

Comment:
do you really need interface for number?

Reply:
refactored id, thanks